addons/website_sale/views/templates.xml

```diff
@@ -967,18 +973,25 @@
                                         </t>
                                     </t>
                                     <p t-if="True" class="css_not_available_msg alert alert-warning">This combination does not exist.</p>
-                                    <div id="o_wsale_cta_wrapper" class="d-flex flex-wrap align-items-center">
+                                    <div id="o_wsale_cta_wrapper" class="d-flex flex-wrap align-items-center flex-grow-1 mb-4">
                                         <t t-set="hasQuantities" t-value="false"/>
                                         <t t-set="hasBuyNow" t-value="false"/>
-                                        <t t-set="ctaSizeBig" t-value="not hasQuantities or not hasBuyNow"/>
 
-                                        <div id="add_to_cart_wrap" t-attf-class="{{'d-none' if combination_info['prevent_zero_price_sale'] else 'd-inline-flex'}} align-items-center mb-2 me-auto">
-                                            <a data-animation-selector=".o_wsale_product_images" role="button" id="add_to_cart" t-attf-class="btn btn-primary js_check_product a-submit flex-grow-1 #{'btn-lg' if ctaSizeBig else ''}" href="#">
-                                                <i class="fa fa-shopping-cart me-2"/>
-                                                <span style="font-weight: bold">ADD TO CART</span>
+                                        <div id="o_price_container_div" t-attf-class="w-100 me-2 {{ '' if combination_info['base_unit_price'] else '' }}">
+                                            <t t-call="website_sale.product_price"/>
+                                            <small t-if="combination_info['base_unit_price']"
+                                                   class="ms-1 me-2 text-muted o_base_unit_price_wrapper d-none" groups="website_sale.group_show_uom_price">
+                                                <t t-call='website_sale.base_unit_price'/>
+                                            </small>
+                                        </div>
+                                        <div id="add_to_cart_wrap" t-attf-class="flex-grow-1 {{'d-none' if combination_info['prevent_zero_price_sale'] else 'd-inline-flex'}}">
+                                            <a data-animation-selector=".o_wsale_product_images"
+                                                role="button" id="add_to_cart"
```

Comment:
Detail: either you move each attribute in a dedicated line or you keep them all in one ;)
Some people do combine short attributes (like `role="button"` and `href="#"` in this case), but I'd advise against.